_layouts/tutorial.html

```diff
@@ -26,6 +26,6 @@
 <h2> Author </h2>
 {% endif %}
 {% for author in page.author %}
-<div style="float: left; margin-right: 20px;">{% include author-card.html author = author %}</div>
+<div style="float: left; margin-right: 20px;">{% include author-card.html author=author theme='light' %}</div>
```

Comment:
Because the background color in the info panel on other pages is lighter than the page background here. The drop shadow didn't give it enough contrast in my opinion to make it easily visible (and shadows don't fit with the other visuals on our site anymore), so I made the card background lighter on dark backgrounds and darker on light backgrounds. The themed highlight on the left side also helps with the contrast.

I called it "theme" just so that it is easy to add other context-specific styles if needed; for example, it might make sense to have a large version and a small version at some point.